Framework/AnalysisSupport/src/AODJAlienReaderHelpers.cxx

```diff
@@ -107,6 +107,12 @@ std::vector<std::string> getColumnNames(header::DataHeader dh)
       return columnNamesTrait(typename StoredTracksCovMetadata::table_t::persistent_columns_t{});
     } else if (description == "TRACK:EXTRA") {
       return columnNamesTrait(typename TracksExtraMetadata::table_t::persistent_columns_t{});
+    } else if (description == "TRACKFWD:PAR") {
+      return columnNamesTrait(typename StoredTracksFwdMetadata::table_t::persistent_columns_t{});
+    } else if (description == "TRACKFWD:PARCOV") {
+      return columnNamesTrait(typename StoredTracksCovMetadata::table_t::persistent_columns_t{});
+    } else if (description == "TRACKFWD:EXTRA") {
+      return columnNamesTrait(typename TracksExtraFwdMetadata::table_t::persistent_columns_t{});
     }
```

Comment:
We should discuss if we need the split into the three tables which adds overhead on reading for a table. Will there be analysis which do not use the covariance matrix? The extra only has 4 fields at present, so is it worth to have it separate?

Reply:
Run2 MUON analysis usually does not uses the covariances matrix. MFTstandalone and GlobalMuonTracks are very certainly to use. As for the extra table, I added three missing columns: 7 fields that I can think of at the moment at the extra table. GlobalMuonTracks may need indexes to the corresponding standalone tracks.

Reply:
The extra table has been incorporated into the main parameters table. Created new table for MFT Standaloneas discussed last week.